src/components/v2/shared/DistributionSplitsSection/DistributionSplitModal.tsx

```diff
@@ -355,62 +429,51 @@ export default function DistributionSplitModal({
               }}
             >
               <FormattedNumberInput
-                value={amount?.toString()}
+                value={amount?.toFixed(4)}
```

Comment:
Because the percents are precise to 8 sig figs, amounts are actually usually '1000.0000001' or something, so need to cut that off. Decided to go to 4dp